pkg/internal/coordinatortest/testdata/propose_sweep_scenario_2.json

```diff
@@ -0,0 +1,43 @@
+{
+  "Title": "Estimated sweep transaction fee greater than max fee",
```

Comment:
I am confused about this test scenario. If I understand correctly what is supposed to happen here is that the fee estimated with `estimateDepositsSweepFee` is higher than the max one allowed by the bridge, and as a result, we cap the fee to the max one allowed.

I am not sure this is valid logic. I would rather log an error and skip proposing the sweep. This is an abnormal situation and I would rather wait until the fee goes back to normal value than block the wallet in the `WalletCoordinator` with something that has no chance of being included in a block. Worth noting, we don't have fee bumps implemented yet.

EDIT: I found the code capping the fee: 

https://github.com/keep-network/keep-core/blob/18b302a8cd934e054c00f8a48d8c0c3babafc00a/pkg/coordinator/sweep.go#L282-L283

I don't think we should block this PR. This is logic from existing code and this PR, by adding this test, revealed this logic. However, I would definitely reconsider this path in a follow-up. Should we return an error from `estimateDepositsSweepFee` in such a case instead? cc @lukasz-zimnoch

Reply:
I added a separate task on the #3614 checklist to track it. This is not that much of a problem for a manual submission because the fee has to be estimated manually first and humans will easily notice when it is absurdly high. For automation, we need to cover this path.